server/simple_server.go

```diff
@@ -110,16 +97,19 @@ func (s *SimpleServer) safelyExecuteRequest(w http.ResponseWriter, r *http.Reque
 			}
 		}
 	}
-	TimedAndCounted(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		if r.Body != nil {
-			defer func() {
-				if err := r.Body.Close(); err != nil {
-					Log.Warn("unable to close request body: ", err)
-				}
-			}()
-		}
-		s.svc.Middleware(s.mux).ServeHTTP(w, r)
-	}), registeredPath, r.Method, s.mets).ServeHTTP(w, r)
+
+	registeredPath = strings.TrimPrefix(registeredPath, "/")
+	prometheus.InstrumentHandler(registeredPath,
```

Comment:
While you're changing this code, can you change this to use `InstrumentHandlerWithOpts ()` with 

```
SummaryOpts{
			Subsystem:   "http",
			ConstLabels: Labels{"handler": handlerName},
			Objectives:  map[float64]float64{0.5: 0.05, 0.9: 0.01, 0.95: 0.005, 0.99: 0.001},
		}
```
This should add the 95%tile which will give us a better comparison number as it seems like a lot of teams use 95% instead of 90%tile or 99%tile.

Danke!

Reply:
This is done. Mind taking a look and verifying its what you want?

Reply:
Looks good - thanks!